python/grass/temporal/register.py

```diff
@@ -226,7 +226,9 @@ def register_maps_in_space_time_dataset(
                 found = gscript.find_file(element=type, name=mapname)
                 if found["mapset"] is not None and len(found["mapset"]) > 0:
```

Comment:
Since you are revising this part anyway, I suggest to change to be more readable:

1. `found -> result`
2. The test can be just `if found["mapset"]` in Python; that's what's written there.

Reply:
Solved in the `build_id_from_search_path` function